Project-1/README.md

```diff
@@ -12,63 +12,296 @@ Tip #4: The work we do is challenging; that should be assumed. You are smart eno
+  B ≡ M ⊓ ∃p2¯.Person
+
   (b)  A that says that x is an aunt of y
+  
+  Answer: 
+  
+  A ≡ ¬M ⊓ (∃p2¯.(≥ 2 parent_of. (∃parent_of. Person)) ⊔ ¬∃parent_of. Person)
+  
   (c)  C that says that x and y are cousins
+  
+  Answer: 
+  
+  gp2 (Grandparent with at least two children which each have children) ≡ ≥2 parentof. (∃parentof. Person)
+  
+  C ≡ gp2¯. Person
+
+  But this would only describe one of the two cousins, i.e. "being a cousin of someone" or "having a cousin". We cannot define "being cousins" in description logic.
+  
+  
   (d)  O that says that x is an only child  
-  (e)  T that says that x has exactly two brothers 
-  ```
+  
+  Answer:
+  
+  parent_only (be a parent of exactly one child) ≡ (≥1 parent_of. Person) ⊓ (≤1 parent_of. Person)
+  O ≡ ∃parent_only¯.Person
+  
+  (e)  T that says that x has exactly two brothers
+  
+  Answer: 
+  
+  p3m (be a parent of exactly three male children) ≡ (≥3 parent_of. Male) ⊓ (≤3 parent_of. Male)
+  p1 (be a parent of at least one child) ≡ ≥1 parent_of. Person -- the inverse of p1 is just "be a child of"!
+  T ≡ ∃p3m¯.Person ⊔ (¬M ⊓ ∃p1¯. (≥2 parent_of. Male ⊓ ≤2 parent_of. Male))
 
 
 5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
 
+Answer: 
+
+K1=(T1,A1) such that:
+
+T-box 
+
+T1={B ≡ M ⊓ ∃p2¯.Person,
+T ≡ ∃p3m¯.Person ⊔ (¬M ⊓ ∃p1¯. (≥2 parent_of. Male ⊓ ≤2 parent_of. Male)),
+Male ⊑ Person,
+Female ⊑ Person,
+Male  ≡ Person ⊓ ¬ Female}
+	
+A-Box: 
+
+{Person(Julia),Person(Mike),Person (Bob), Person (Paul),
+Male(Mike), Male (Bob),
+Female (Julia),
+B(Julia), B(Bob), B(Mike),
+T(Mary),
+Parent_of(Paul, Mike), Parent_of(Paul,Mary), Parent_of(Paul,Bob)}
+
+
 6. Explain the difference - using natural language - between the first-order prefixes:
-  ```
+  
   (a) ∃x∀y and ∀x∃y
+  Answer: the former means that there exists a x that for all y such that, while the latter means that for all x there exists a y such that.
+  
   (b) ∃x∀y∃z and ∀x∃y∀z 
+  Answer: the former means that there exists a x that for all y there exists a z such that, while the latter means that for all x there exists a y that for all z such that. 
+  
   (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
-```
+  Answer: the former means that for all x, there exists a y that for all z, there exists a w such that, while the latter means that there exists a x that for all y, there exists a z that for all w such that.
+  
 	
 7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
-```
+
 ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
 ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
-```
+
+Answer: 
+
+The following graph models the second sentence but not the first sentence:
+Verticles: a, b, c
+Edges: ab, ac, aa (loop)
+
+See attached picture: image 0.
```

Comment:
Please express the graph in mermaid in the README. Do not include separate image files.